src/modules/material-requests/types/index.ts

```diff
@@ -135,6 +148,9 @@ export enum MaterialRequestKeys {
 	createdAt = 'createdAt',
 	updatedAt = 'updatedAt',
 	type = 'type',
+	status = 'status',
+	download = 'downloadUrl',
```

Comment:
maybe keep the enum in sync with the value
```suggestion
	downloadUrl = 'downloadUrl',
```